### DISCUSSION TITLE
Please review my Harvest install script

### DISCUSSION THREAD

**Discussion by alex-harvey-z3q** [NetApp/harvest#2832] [General]
Hi all, I am finding the docs and set up quite confusing but I _think_ something like this should work:
https://gist.github.com/alexharv074/6992854b7c6847396f08074d6cf44649

**Comment by rahulguptajss** [comment]
@alexharv074 The script appears to be fine. Could you share the error that you're encountering? Also, please check if there are no issues related to the docker pull command.
